gems/aws-sdk-dynamodbstreams/lib/aws-sdk-dynamodbstreams/attribute_value.rb

```diff
@@ -0,0 +1,117 @@
+# frozen_string_literal: true
+
+require 'bigdecimal'
+require 'stringio'
+require 'set'
+
+module Aws
+  module DynamoDBStreams
+    # @api private
+    class AttributeValue
+
+      def initialize
+        @marshaler = Marshaler.new
+        @unmarshaler = Unmarshaler.new
+      end
+
+      def marshal(value)
+        @marshaler.format(value)
+      end
+
+      def unmarshal(value)
+        @unmarshaler.format(value)
+      end
+
+      class Marshaler
```

Comment:
In theory we could do away with this here since its unused.  But for the purposes of keeping it the same as the DDB one, I think its better to leave.

Reply:
I thought it was fine to keep. Maybe it makes sense to remove.